src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTable/MyProfileCounterpartiesTable.scss

```diff
@@ -1,4 +1,5 @@
 .my-profile-counterparties-table {
+    height: unset !important;
```

Comment:
I need to override the style set here for the table, if i remove this, this will affect all the other tables, hence why i need !important

![Screenshot 2024-08-13 at 1 46 45 PM](https://github.com/user-attachments/assets/c0770097-bfb7-4f6e-96e1-cd5dc9401689)

Reply:
why not add this 
```
 & .table {
            height: unset;
        }
```

under my-profile-counterparties__content ?

Reply:
we could do that, but because we can pass classNames to the table itself, we don't need to directly access the table from here, and we can just change the styles from the className passed to the table. But the end result is the same

Reply:
@ameerul-deriv let's get rid of `important` as much as possible.